docs/src/pages/docs/functions/combinators.md

```diff
@@ -20,6 +20,32 @@ give it a value and it will give you back a function ready to take a function.
 Once that function is provided, it will return the result of applying your value
 to that function.
 
+```javascript
+import applyTo from 'crocks/combinators/applyTo'
+
+import map from 'crocks/pointfree/map'
+import compose from 'crocks/helpers/compose'
+
+// prices :: [ Number ]
+const prices = [4.99, 29.99, 15.99];
```

Comment:
for cleanliness, lets add spaces  after [ and before ]

Reply:
Also somehow a semi-colon was able to be committed.